test/e2e/marketplace_test.go

```diff
@@ -32,14 +28,14 @@ func TestMarketplace(t *testing.T) {
 		TypeMeta: metav1.TypeMeta{
 			Kind: marketplace.OperatorSourceKind,
 			APIVersion: fmt.Sprintf("%s/%s",
-			marketplace.SchemeGroupVersion.Group, marketplace.SchemeGroupVersion.Version),
+				marketplace.SchemeGroupVersion.Group, marketplace.SchemeGroupVersion.Version),
```

Comment:
Formatting fixes should be a separate PR

Reply:
This was a mistake introduced by the go format when saving the file, I will revert the change.